sfge/build.gradle.kts

```diff
@@ -49,6 +49,7 @@ application {
 spotless {
 	java {
 		googleJavaFormat().aosp().groupArtifact("com.google.googlejavaformat:google-java-format")
+		toggleOffOn()
```

Comment:
Allows me to use `// spotless:off` and `// spotless:on` to enclose code I don't want getting reformatted. See https://github.com/diffplug/spotless/tree/main/plugin-gradle#spotlessoff-and-spotlesson for more detail.